pkg/operator/apis/monitoring/v1alpha1/types_logs.go

```diff
@@ -153,8 +153,6 @@ type LogsInstanceList struct {
 }
 
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:path="pod-logs"
-// +kubebuilder:resource:singular="pod-logs"
```

Comment:
The hyphens really aren't how k8s expects you to refer to multi-word names (daemonsets, not daemon-sets). I only changed this for podlogs but I suspect we'll want to do the same for grafana-agent, prometheus-instance and logs-instance in a later PR.